File: Examples/CSharp/Worksheets/PageSetupFeatures/PageOrientation.cs
using System.IO;
using Aspose.Cells;
using System;

namespace Aspose.Cells.Examples.CSharp.Worksheets.PageSetupFeatures
{
    public class PageOrientation
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            // Instantiating a Workbook object
            Workbook workbook = new Workbook();

            // Accessing the first worksheet in the Excel file
            Worksheet worksheet = workbook.Worksheets[0];

            // Setting the orientation to Portrait
            worksheet.PageSetup.Orientation = PageOrientationType.Portrait;

            // Save the Workbook.
            workbook.Save(dataDir + "PageOrientation_out.xls");
            // ExEnd:1
        }
    }
}

File: Examples/VisualBasic/Worksheets/PageSetupFeatures/PageOrientation.vb
Imports System.IO
Imports Aspose.Cells

Namespace Worksheets.PageSetupFeatures
    Public Class PageOrientation
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Instantiating a Workbook object
            Dim workbook As New Workbook()

            ' Accessing the first worksheet in the Excel file
            Dim worksheet As Worksheet = workbook.Worksheets(0)

            ' Setting the orientation to Portrait
            worksheet.PageSetup.Orientation = PageOrientationType.Portrait

            ' Save the Workbook.
            workbook.Save(dataDir & Convert.ToString("PageOrientation_out.xls"))
            ' ExEnd:1
        End Sub
    End Class
End Namespace
